app/models/post.rb

```diff
@@ -10,4 +10,7 @@ def update_posts_counter!
   def five_recent_comments
     comments.order(created_at: :desc).limit(5)
   end
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Currently, your `post_spec.rb` is also missing tests for these two validations and the two methods in your `Post Model` so kindly include tests for them as well :wink: